example-library/src/main/java/uk/gov/gchq/palisade/example/library/common/EmployeeUtils.java

```diff
@@ -44,16 +44,11 @@ public static boolean isManager(final Manager[] managers, final UserId userId) {
         }
 
         for (Manager manager : managers) {
-            if (manager.getUid().equals(userId.getId())) {
+            if (manager.getUid().equals(userId.getId()) || isManager(manager.getManager(), userId)) {
                 return true;
             }
         }
 
-        for (Manager manager : managers) {
-            if (isManager(manager.getManager(), userId)) {
-                return true;
-            }
-        }
         return false;
```

Comment:
These changes aren't semantically equivalent, and depending on what the tree of managers looked like there might be a preference to do  one way or the other.
But - this is an example and it doesn't really matter...

Reply:
SQ thought these for loops should be collated into 1. I might try and rework this back into separate loops.